File: python/uberVan.py
from car import Car

class UberVan(Car):
    typeCarAccepted = []
    seatsMaterial   = []
    
    def __init__(self, license, driver, typeCarAccepted, seatsMaterial):
        super().__init__( license, driver)
        self.seatsMaterial = seatsMaterial
        self.typeCarAccepted = typeCarAccepted

File: java/UberVan.java
import java.util.ArrayList;
import java.util.Map;

class UberVan extends Car {
    Map<String, Map<Integer, String>> typeCarAccepted;
    ArrayList<Integer>  seatsMaterial;
    
    public UberVan(String license, Account driver) {
        super(license, driver);
    }

    public UberVan(String license, Account driver, Map<String, Map<Integer, String>> typeCarAccepted,
            ArrayList<Integer> seatsMaterial) {
        super(license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }

    @Override
    public void setPassenger(Integer passenger) {
        if (passenger == 6) {
            System.out.println("Passengers : " + passenger);
            }else{
               System.out.println("It's onlie 6 passenger");
            }
    }
    
}
